serotiny/io/image.py

```diff
@@ -84,6 +86,14 @@ def image_loader(
         If reading an ome zarr image, this specifies the image name
     """
 
+    # silence aicsimageio related warnings
+    warnings.filterwarnings(action="ignore", category=FutureWarning, module="ome_types")
+    logging.getLogger("xmlschema").setLevel(logging.ERROR)
+    logging.getLogger("bfio").setLevel(logging.ERROR)
+    logging.getLogger("bfio.backends").setLevel(logging.ERROR)
+    logging.getLogger("ome_zarr").setLevel(logging.ERROR)
+    logging.getLogger("ome_zarr.reader").setLevel(logging.ERROR)
```

Comment:
I think that the behavior that was generating output here was fixed in the most recent version of ome-zarr-py, so I'd rather bump the ome-zarr-py version in our requirements